utils/icons.js

```diff
@@ -44,3 +44,60 @@ export function HouseSvg({ isFilled }) {
     </>
   );
 }
+
+export function PuzzleElement() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="19"
+      height="16"
+      fill="none"
+      viewBox="0 0 19 16"
+    >
+      <mask id="b" fill="#fff">
+        <path
+          fill-rule="evenodd"
```

Comment:
in Next.js using styled components the naming convention is fillRule or clipRule.
You can use the CLI to convert all of your icons with svgr. removing fills and convert everything, as it needs.

Here is a link for a tutorial i used.

[Youtube-Link](https://www.youtube.com/watch?v=v0ZLEy1SE-A&t=882s&ab_channel=MykhayloRyechkin)

Reply:
Thanks ✌️